packages/kit/src/components/Hardware/Hardware.tsx

```
@@ -526,8 +526,8 @@ export function EnterPassphraseOnDevice({
           '@onekeyhq/kit/assets/animations/enter-passphrase-on-pro-dark.json'
         );
       default:
-        // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-case-declarations
-        const checkType: never = deviceType;
+        // Exhaustiveness check for TypeScript
+        const _exhaustiveCheck: never = deviceType;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)